packages/ui/src/working-groups/components/Roles/RolesList.tsx

```diff
@@ -31,18 +35,28 @@ export interface RolesListProps {
   workers: Worker[]
 }
 
-export const RolesList = ({ workers }: RolesListProps) => (
-  <List>
-    {workers.map((worker) => (
-      <ListItem key={worker.id} borderless>
-        <RolesListItem worker={worker} />
-      </ListItem>
-    ))}
-  </List>
-)
+export const RolesList = ({ workers }: RolesListProps) => {
+  const blockNumber = useCurrentBlockNumber()
+  const { api } = useApi()
+  const nextPayout = useMemo(
+    () => blockNumber && getNextPayout(workers, blockNumber, api),
+    [workers.length, blockNumber?.toNumber()]
```

Comment:
'Next payment in' 0 JOY => leave box on `RolesListItem` empty if zero
That total paid is zero may be unrelated?

![bwg-job](https://user-images.githubusercontent.com/31551045/212012372-da09c7b0-4f63-4763-8aaf-ecf80b184154.png)

I suggest to rename the (passed) variable to `nextPayment` and adjust `RolesListItem` accordingly.